src/odbc_writer.rs

```diff
@@ -80,7 +80,8 @@ fn insert_statement_text(table: &str, column_names: &[&'_ str]) -> String {
 
 /// Wraps column name in quotes, if need be
 fn quote_column_name(column_name: &str) -> Cow<'_, str> {
-    if column_name.contains(|c| !valid_in_column_name(c)) {
+    if column_name.contains(|c| !valid_in_column_name(c)) 
+        || PROTECTED_KEYWORDS_LOWER.contains(&column_name.to_lowercase().as_str()) {
```

Comment:
I would prefer case insensitive comparison over allocating new strings. Could we use `eq_ignore_ascii_case` here? See: <https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case>